inlong-sort/sort-formats/format-common/src/main/java/org/apache/inlong/sort/formats/util/StringUtils.java

```diff
@@ -97,133 +97,135 @@ public static List<Map<String, String>> splitKv(
-                            stringBuilder.append(lastCh);
-                        }
-                        kvState = state;
-                        state = STATE_ESCAPING;
-                        break;
-                    case STATE_VALUE:
-                        kvState = state;
-                        state = STATE_ESCAPING;
-                        break;
-                    case STATE_ESCAPING:
-                        stringBuilder.append(ch);
-                        state = kvState;
-                        break;
-                    case STATE_QUOTING:
-                        stringBuilder.append(ch);
-                        break;
-                }
-            } else if (quoteChar != null && ch == quoteChar) {
-                switch (state) {
-                    case STATE_KEY:
-                        if (lastCh != 0) {
-                            stringBuilder.append(lastCh);
-                        }
-                        kvState = state;
-                        state = STATE_QUOTING;
-                        break;
-                    case STATE_VALUE:
-                        kvState = state;
-                        state = STATE_QUOTING;
-                        break;
-                    case STATE_ESCAPING:
-                        stringBuilder.append(ch);
-                        state = kvState;
-                        break;
-                    case STATE_QUOTING:
-                        state = kvState;
-                        break;
-                }
-            } else if (lineDelimiter != null && ch == lineDelimiter) {
-                switch (state) {
-                    case STATE_KEY:
-                        key = lastKey;
-                        stringBuilder.append(lastValue).append(lastCh);
-                        value = stringBuilder.toString();
-                        fields.put(key, value);
-                        lastKey = null;
-                        lastValue = null;
-                        stringBuilder.setLength(0);
-                        break;
-                    case STATE_VALUE:
-                        lastKey = null;
-                        lastValue = null;
-                        value = stringBuilder.toString();
-                        fields.put(key, value);
-                        Map<String, String> copyFields = new HashMap<>();
-                        copyFields.putAll(fields);
-                        lines.add(copyFields);
-                        stringBuilder.setLength(0);
-                        fields.clear();
-                        state = STATE_KEY;
-                        break;
-                    case STATE_ESCAPING:
-                        stringBuilder.append(ch);
-                        state = STATE_NORMAL;
-                        break;
-                    case STATE_QUOTING:
-                        stringBuilder.append(ch);
-                        break;
+        if (text != null) {
```

Comment:
If text == null, then it is better to return lines directly.

Reply:
fixed.